### ISSUE TITLE
"cpu fp32i8" strategy not working in RWKV v6 through Python rwkv module

### ISSUE THREAD

**Issue by polkovnikov** [BlinkDL/ChatRWKV#193]
When I use PIPELINE, PIPELINE_ARGS from rwkv.utils together with current RWKV v6 checkpoint `rwkv-x060-1b6-world-v2-68%trained-20231208-ctx4k.pth` I get following error in pipeline.generate() with `cpu fp32i8` strategy:

```
  File "C:\bin\Python39\lib\site-packages\rwkv\utils.py", line 97, in generate
    out, state = self.model.forward(tokens[:args.chunk_len], state)
  File "C:\bin\Python39\lib\site-packages\rwkv\model.py", line 1152, in forward
    x, state[i*3+0], state[i*3+1] = ATT(
RuntimeError: The following operation failed in the TorchScript interpreter.
Traceback of TorchScript (most recent call last):
  File "C:\bin\Python39\lib\site-packages\rwkv\model.py", line 869, in att_seq_v6_0
        sx = torch.cat((sx.unsqueeze(0), xx[:-1,:])) - xx
        xxx = xx + sx * x_maa
        xxx = torch.tanh(xxx @ tm_w1).view(T, 5, -1).transpose(0, 1)
                         ~~~~~~~~~~~ <--- HERE
        xxx = torch.bmm(xxx, tm_w2).view(5, T, -1)
        mw, mk, mv, mr, mg = xxx.unbind(dim=0)
RuntimeError: expected m1 and m2 to have the same dtype, but got: float != unsigned char
```

same generate() code works well for RWKV v5 checkpoint for some test string context.

Error above looks to me that it is issue of `fp32i8`, because unsigned char means i8 quantized number and float means float-32 number.

RWKV v6 model downloaded from [https://huggingface.co/BlinkDL/temp/](https://huggingface.co/BlinkDL/temp/).

Full code that gave issue:

```python
import os

os.environ['RWKV_JIT_ON'] = '1'
os.environ["RWKV_CUDA_ON"] = '0' # '1' to compile CUDA kernel (10x faster), requires c++ compiler & cuda libraries

from rwkv.model import RWKV
from rwkv.utils import PIPELINE, PIPELINE_ARGS

# downloaded model from: https://huggingface.co/BlinkDL

model = RWKV(model='./rwkv-x060-1b6-world-v2-68%trained-20231208-ctx4k', strategy='cpu fp32i8')
pipeline = PIPELINE(model,
    'rwkv_vocab_v20230424',
)

ctx = "\nIn a shocking finding, scientist discovered a herd of dragons living in a remote, previously unexplored valley, in Tibet. Even more surprising to the researchers was the fact that the dragons spoke perfect Chinese."
print(ctx, end='')

def my_print(s):
    print(s, end='', flush=True)

args = PIPELINE_ARGS(
    temperature = 0.05,
    top_p = 0.7, top_k = 100, # top_k = 0 then ignore
    alpha_frequency = 0.25,
    alpha_presence = 0.25,
    alpha_decay = 0.996, # gradually decay the penalty
    token_ban = [0], # ban the generation of some tokens
    token_stop = [], # stop generation whenever you see any token here
    chunk_len = 256,
) # split input into chunks to save VRAM (shorter -> slower)

pipeline.generate(
    ctx, token_count=200,
    args=args,
    callback=my_print)
print('\n')
```

**Comment by BlinkDL** [comment]
fixed in 0.8.25 :)

**Comment by polkovnikov** [comment]
@BlinkDL Verified, works for me! Closing issue.
